R/derive_var_ady.R

```diff
@@ -41,14 +41,5 @@
 #' @export
 #'
 derive_var_ady <- function(dataset, reference_date = TRTSDT, date = ADT) {
-  reference_date <- assert_symbol(enquo(reference_date))
-  date <- assert_symbol(enquo(date))
-  assert_data_frame(dataset, vars(!!reference_date, !!date))
-  deprecate_warn("0.7.0", "derive_var_ady()", "derive_vars_dy()")
-
-  derive_vars_dy(
-    dataset,
-    reference_date = !!reference_date,
-    source_vars = vars(ADY = !!date)
-  )
+  deprecate_stop("0.8.0", "derive_var_ady()", "derive_vars_dy()")
```

Comment:
I actually think the first argument should stay at 0.7.0. According to the documentation the `when` argument is a "string giving the version when the behaviour was deprecated".

Reply:
OK I see, I thought it was the version when it was moved to a certain stage of the deprecation status.

Reply:
Fixed.